coops-ui/public/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
-    <title>Cooperatives</title>
+    <title>Cooperatives Online</title>
```

Comment:
Totally inconsequential, but why this change?

Reply:
Per Jeremy. Part of the branding/marketing of our apps.

Reply:
Very cool, happy to hear we're `Online` now 😄

Reply:
Hahaha